mitodl/mitxpro
# #679 Set an HTML title on React pages

#### Pre-Flight checklist


#### What are the relevant tickets?
#679 

#### What's this PR do?
Sets appropriate HTML titles on React rendered pages.

#### How should this be manually tested?
Go to any page that is rendered by React (login/enrollment/profile etc). The title of the document should change dynamically on each page.

#### Screenshots (if appropriate)
![Screenshot from 2019-07-16 16-41-04](https://user-images.githubusercontent.com/45350418/61291736-cc19b980-a7e8-11e9-9242-b12a6af68a81.png)
![Screenshot from 2019-07-16 16-40-01](https://user-images.githubusercontent.com/45350418/61291737-cc19b980-a7e8-11e9-9de1-d4e722ae8438.png)
![Screenshot from 2019-07-16 16-34-31](https://user-images.githubusercontent.com/45350418/61291739-ccb25000-a7e8-11e9-8e4b-62303a27e025.png)
